tutorial/Tutorial.md

```diff
@@ -0,0 +1,78 @@
+# Move Pallet Tutorial
+
+## Table of Contents
+
+- [Overview](#overview)
+- [The Code Example](#the-code-example)
+- [Publishing the Module](#publishing-the-module)
+- [Estimation of Needed Gas](#estimation-of-needed-gas)
+- [Demonstration of Use Case](#demonstration-of-use-case)
+
+## Overview
+
+Move is a smart-contract language with focus on security and the possibility to write your individual resources. To make it interoperable with Substrate we provide the package manager `smove`, which can compile your Move projects, create bundles, access and act with the network node and more. Like in Rust we have Cargo, in Move we have [`smove`](https://github.com/eigerco/smove).
```

Comment:
```suggestion
Move is a smart contract language with a focus on security and the possibility of writing your individual resources. To make it interoperable with Substrate, we provide the package manager `smove`, which can compile your Move projects, create bundles, access and act with the network node and more. Like in Rust we have Cargo, in Move we have [`smove`](https://github.com/eigerco/smove).
```

Reply:
Done, as far as possible, there were more modifications before due to requests from Karlo :blush: